src/explore-education-statistics-admin/src/pages/legacy-releases/components/LegacyReleasesTable.tsx

```diff
@@ -1,83 +1,62 @@
 import ButtonLink from '@admin/components/ButtonLink';
-import Page from '@admin/components/Page';
-import usePublicationContext from '@admin/contexts/PublicationContext';
 import {
   legacyReleaseCreateRoute,
   legacyReleaseEditRoute,
 } from '@admin/routes/legacyReleaseRoutes';
-import { dashboardRoute, PublicationRouteParams } from '@admin/routes/routes';
+import { PublicationRouteParams } from '@admin/routes/routes';
 import legacyReleaseService, {
   LegacyRelease,
 } from '@admin/services/legacyReleaseService';
-import publicationService from '@admin/services/publicationService';
-import appendQuery from '@admin/utils/url/appendQuery';
+import publicationService, {
+  BasicPublicationDetails,
+} from '@admin/services/publicationService';
 import Button from '@common/components/Button';
 import ButtonGroup from '@common/components/ButtonGroup';
-import LoadingSpinner from '@common/components/LoadingSpinner';
 import ModalConfirm from '@common/components/ModalConfirm';
 import useToggle from '@common/hooks/useToggle';
 import reorder from '@common/utils/reorder';
+import styles from '@admin/pages/legacy-releases/components/LegacyReleasesTable.module.scss';
 import classNames from 'classnames';
 import React, { useState } from 'react';
 import { DragDropContext, Draggable, Droppable } from 'react-beautiful-dnd';
 import { generatePath } from 'react-router';
-import styles from './LegacyReleasesPage.module.scss';
-
-const LegacyReleasesPage = () => {
-  const {
-    value: publication,
-    setState: setPublication,
-    isLoading,
-    retry: reloadPublication,
-  } = usePublicationContext();
 
+interface Props {
+  publication: BasicPublicationDetails;
+}
+const LegacyReleasesTable = ({ publication }: Props) => {
   const [isReordering, toggleReordering] = useToggle(false);
   const [deleteLegacyRelease, setDeleteLegacyRelease] = useState<
     LegacyRelease
   >();
 
-  if (!publication) {
-    return <LoadingSpinner loading={isLoading} />;
-  }
+  const [legacyReleases, setLegacyReleases] = useState<LegacyRelease[]>(
```

Comment:
I *think* that you don't need the `<LegacyRelease[]>` part here, as useState can infer its type based upon you passing `publication.legacyReleases` into it.

Reply:
You are correct :)